Solutions/GreyNoiseThreatIntelligence/Data Connectors/GreyNoiseConnector_UploadIndicatorsAPI.json

```diff
@@ -0,0 +1,124 @@
+        },
+        {
+            "title": "1. Retrieve API Key from GreyNoise Portal.",
+            "description": "Generate an API key from GreyNoise Portal https://docs.greynoise.io/docs/using-the-greynoise-api"
+        },
+        {
+            "title": "2. In your Azure AD tenant, create an Azure Active Directory (AAD) application and acquire Tenant ID, Client ID and Client Secret.",
+            "description": "Follow the instructions here to create your Azure AAD app and save your Client ID, Client Secret and Tenant ID: https://learn.microsoft.com/en-us/azure/sentinel/connect-threat-intelligence-upload-api#instructions\n NOTE: Wait until step 5 to generate your client secret."
+        },
+        {
+            "title": "3. Assign the AAD application the Microsoft Sentinel Contributor Role.",
+            "description": "Follow the instructions here to add the Microsoft Sentinel Contributor Role: https://learn.microsoft.com/en-us/azure/sentinel/connect-threat-intelligence-upload-api#assign-a-role-to-the-application"
+        },
+        {
+            "title": "4. Specify the AAD permissions to enable MS Graph API access to the upload-indicators API.",
+            "description": "Follow this section here to add 'ThreatIndicators.ReadWrite.OwnedBy' permission: Client Secret and Tenant ID: https://learn.microsoft.com/en-us/azure/sentinel/connect-threat-intelligence-tip#specify-the-permissions-required-by-the-application"
+        },{
+            "title": "5. Deploy the Threat Intellegence (Preview) Solution which includes the Threat Intelligence Upload Indicators API (Preview)",
+            "description": "See content hub for this Solution."
+        },
+        {
+            "title": "6. Deploy the Azure Function",
+            "description": "Click the Deploy to Azure button below  [![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fa531176934d67267585713f73a777cedb3bbc4c1%2FSolutions%2FGreyNoise%2FData%2520Connectors%2Fazuredeploy_Connector_GreyNoiseAPISentinel_AzureFunction.json)\n"
+        },
+        {
+            "title": "7. Send indicators to Sentinel",
+            "description": "You can send indicators by calling our Upload Indicators API. For more information about the API, click here. \n\n>HTTP method: POST \n\n>Endpoint: https://sentinelus.azure-api.net/workspaces/{WORKSPACEID}/threatintelligenceindicators:upload?api-version=2022-07-01  \n\n>WORKSPACEID: the workspace that the indicators are uploaded to  (can be copied from the following).  \n\n\n>Header Value 1: \"Authorization\" = \"Bearer [AAD Access Token from step 1]\" \n\n\n> Header Value 2: \"Content-Type\" = \"application/json\"  \n \n>Body: The body is a JSON object containing an array of indicators in STIX format. For more information about the API, click here",
```

Comment:
Hello @punkrokk, You have mentioned [ click here ] twice in this description but didn't added a hyperlink to it.
And also after hitting the endpoint link its throwing 404. Is it ok?

Reply:
Fixed